main/src/modules/systemVisual/iol2opc/src/module.cpp

```diff
@@ -196,27 +196,52 @@ bool IOL2OPCBridge::getClickPosition(CvPoint &pos)
 /**********************************************************/
 bool IOL2OPCBridge::get3DPosition(const CvPoint &point, Vector &x)
 {
+    x.resize(3,0.0);
     if (rpcGet3D.getOutputCount()>0)
     {
+        // thanks to SFM we are here
+        // safe against borders checking
         Bottle cmd,reply;
-        cmd.addString("Root");
-        cmd.addInt(point.x);
-        cmd.addInt(point.y);
+        cmd.addString("Rect");
+        cmd.addInt(point.x-3);
+        cmd.addInt(point.y-3);
+        cmd.addInt(7);
+        cmd.addInt(7);
+        cmd.addInt(2);
+
```

Comment:
I think it would make sense putting a small comment on what those magic numbers mean. It's clear from the `SFM` documentation, but needing to look it up is cumbersome. Should we just copy the description from `SFM` here?

> [Rect tlx tly w h step]: Given the pixels in the rectangle defined by {(tlx,tly) (tlx+w,tly+h)} (parsed by columns), the response contains the corresponding 3D points in the ROOT frame. The optional parameter step defines the sampling quantum; by default step=1.